commit 3ff08caa35db539fcc3dded353ec03c9f6a6efe7
Author: Corentin Wallez <cwallez@chromium.org>
Date:   Mon Oct 14 21:05:42 2024 +0000

    StringViewUtils.cpp: Add include for std::strlen.
    
    Bug: 42241188
    Change-Id: I8950589cb51987d385d179b05655caeabf9462a6
    Reviewed-on: https://dawn-review.googlesource.com/c/dawn/+/210755
    Reviewed-by: Loko Kung <lokokung@google.com>
    Commit-Queue: Loko Kung <lokokung@google.com>

--- a/third_party/dawn/src/dawn/common/StringViewUtils.cpp
+++ b/third_party/dawn/src/dawn/common/StringViewUtils.cpp
@@ -26,6 +26,7 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "dawn/common/StringViewUtils.h"
+#include <cstring>
 
 bool operator==(WGPUStringView a, WGPUStringView b) {
     return wgpu::StringView(a) == wgpu::StringView(b);
